.gitignore

```diff
@@ -82,3 +82,4 @@ dist
 
 # IDE
 .idea
+.DS_Store
```

Comment:
Could you also add storybook-static here? That is the default directory for output from `npm run build-storybook`

Reply:
Yes, it is in the gitignore now.